indico/modules/events/abstracts/templates/management/_abstract_list.html

```diff
@@ -51,6 +52,36 @@
     </div>
 {% endmacro %}
 
+{% macro _render_proposed_contrib_types(review_list) %}
+    {%- for type, reviews in review_list|groupby('proposed_contribution_type') %}
```

Comment:
{%- for type, reviews in review_list|groupby('proposed_contribution_type') if type %}

and remove the separate `if` below